transfluent.py

```diff
@@ -230,12 +230,15 @@ def file_save(self, identifier, language, file, type, format='UTF-8',
         try:
             content = file.read()
         except AttributeError:
-            content = str(file)
+            try:
+                content = bytes(file, format)
+            except TypeError:
+                content = str(file)
         data = {
             'identifier': identifier,
             'language': language,
             'format': format,
-            'content': base64.b64encode(content),
+            'content': base64.b64encode(content).decode(format),
```

Comment:
decode using `'ascii'` encoding instead of `format`. Base64 encoded strings are always ASCII.